code/game/objects/effects/portals.dm

```diff
@@ -120,7 +120,7 @@
 /obj/effect/portal/spawner/phoron
 	num_of_spawns = 3
 	spawn_things = list(
-				"/obj/item/stack/material/phoron" = 10
+				"/obj/item/stack/material/phoron" = 3
```

Comment:
Undocumented change? Does this relate to the slime changes you talked about?

Reply:
this is to go along with the phoron scarcity arc the lore team has going on, so i'd like to not put it in the changelog and be it something players feel, rather than know for sure